src/components/auth/RegisterAgreement.tsx

```diff
@@ -54,12 +55,42 @@ export const RegisterAgreement = ({
     toPageNext();
   };
 
+  const screenChange = (event: { matches: boolean }) => {
+    const matches: boolean = event.matches;
```

Comment:
가져올 객체 필드의 이름과 할당할 변수의 이름이 같은 경우에는 객체 분해 문법을 사용하면 더 깔끔하고 가독성이 좋습니다.

```suggestion
    const { matches } = event;
```

ref: https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment